examples/data_collect/main.cpp

```diff
@@ -95,6 +98,7 @@ int main(int argc, char *argv[]) {
         {"-s", {"--split", false, "", "", false}},
         {"-t", {"--netlinktest", false, "", "", false}},
         {"-st", {"--singlethread", false, "", "", false}},
+        {"-c", {"--c", false, "", "", false}},
```

Comment:
We currently have -config. Adding -c which stands for configuration can be highly misleading. We should look for a better name.